package.json

```diff
@@ -47,7 +47,8 @@
     "ffjavascript": "0.2.55",
     "js-sha3": "^0.8.0",
     "logplease": "^1.2.15",
-    "r1csfile": "0.0.36"
+    "r1csfile": "0.0.40",
+    "readline": "^1.3.0"
```

Comment:
```suggestion
```
I've removed this dependency because it is not correct to include it.

Reply:
done!

Reply:
Thanks for cleaning that up! ❤️